### ISSUE TITLE
Variable to control the spacing of the form elements.

### ISSUE THREAD

**Issue by crabnky** [jgthms/bulma#1587]
This is about Bulma
Is it a feature
I'm using Bulma 0.6.1 

I want to control spacing between the form elements. For now the vertical and horizontal spacing is set to 0.75rem and I have to overwrite styles if I want to change it.
Generally the styles are margin-bottom and margin-right added to .field class or .control if under .field.grouped.

I think it would be useful to have a variable to set this.

Thanks.

**Comment by lokiloq** [comment]
> This is about Bulma
> Is it a feature
> I'm using Bulma 0.6.1
> 
> I want to control spacing between the form elements. For now the vertical and horizontal spacing is set to 0.75rem and I have to overwrite styles if I want to change it.
> Generally the styles are margin-bottom and margin-right added to .field class or .control if under .field.grouped.
> 
> I think it would be useful to have a variable to set this.
> 
> Thanks.

I agree, the default form elements spacing (margin-botton 0.75rem) is not enough.
Would be great to be able to customize it :)

**Comment by kasperkamperman** [comment]
Indeed a variable would be useful. I expected that `$control-padding-vertical` and `$control-padding-horizontal` had this function, but this is padding inside the control.
